scripts/functions.ahk

```diff
@@ -334,11 +332,7 @@ ExitWorld(nextInst:=-1)
     FileDelete,%holdFile%
     FileDelete, %killFile%
     SetAffinities(nextInst)
-    if f1States[idx] ; goofy ghost pie removal
```

Comment:
use this instead:

    if f1States[idx]
      ControlSend,, {Blind}{F1}{F3}{Esc 3}, ahk_pid %pid%
    else
      ControlSend,, {Blind}{F3}{Esc 3}, ahk_pid %pid%